### PR TITLE
Added sendCommandWithCode to enable use of custom rollingCode storage

### PR DESCRIPTION
**Pull request by Mirdinus**
Hi,
I'm following up on my previous PR ([#28](https://github.com/Legion2/Somfy_Remote_Lib/pull/28))

I have added `sendCommandWithCode` method to allow external storage of rolling codes.

I had a problem with NVS on ESP32C3 and decided to keep the rolling codes "offloaded". This approach allows users to use this library on bridge type of devices where there is no need to keep state of the controlled shutters.

### CODE REVIEW

File: src/SomfyRemote.h [hunk 1]
```diff
@@ -40,6 +40,15 @@ class SomfyRemote {
 	 * 				 only be used when simulating holding a button.
 	 */
 	void sendCommand(Command command, int repeat = 4);
+	/**
+	 * Send a command with this SomfyRemote.
+	 *
+	 * @param command the command to send
+	 * @param repeat the number how often the command should be repeated, default 4. Should
+	 * 				 only be used when simulating holding a button.
+	 * @param rollingCode the rolling code to use, default 0. Should only be used with external storage.
```
Comment (Legion2) [inline src/SomfyRemote.h:49]: please reorder the parameter in the documentation to match the function signature, also there is no default value for rollingCode

**Review by Legion2 — CHANGES_REQUESTED** [review]
Thanks for your contribution

File: src/SomfyRemote.h (hunk 1 above)
Comment (Mirdinus) [reply src/SomfyRemote.h]: Done. Thank for catching the default value. I was reusing code from PR #28 and missed that

**Review by Legion2 — APPROVED** [review]

**Comment by Mirdinus** [comment]
> Can you squash your commits please

Done

**Pull request merged**